.github/workflows/release.yml

```diff
@@ -8,11 +8,11 @@ on:
 jobs:
   publish-releases:
     name: Publish Releases
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Running on a moving target is going to make this prone to failure. If the action fix works, then let's keep the workflow container image constant.